nss-cohort-37/Workforce-Blue-Barracudas
# Jw employees crud

# Description
CRUD Complete on Employee Table, Employees can be created and edited including assigning computers.
Closes #2 #3 #4 
## Type of change
Please delete options that are not relevant.
# Testing Instructions
- Click create new on index view of Employees, complete form fields, click create button to submit. New employee should appear in Index view.
- Click Edit link next to an employee, change fields, click edit button. Edited employee should appear in index view and changes should be reflected in employee's details.
# Checklist:
